pkg/qemu-ga/Makefile

```diff
@@ -0,0 +1,14 @@
+.PHONY: tag push
+
+ORG?=linuxkit
+IMAGE=qemu_ga
```

Comment:
Ah yes, that's because I created the directory but then found $something which didn't allow `-`.